Formula/box2d.rb

```diff
@@ -1,11 +1,23 @@
 class Box2d < Formula
   desc "2D physics engine for games"
   homepage "https://box2d.org"
-  url "https://github.com/erincatto/box2d/archive/v2.4.1.tar.gz"
-  sha256 "d6b4650ff897ee1ead27cf77a5933ea197cbeef6705638dd181adc2e816b23c2"
   license "MIT"
+  revision 1
```

Comment:
What user-facing changes are there here?

Reply:
We aren't including the `doctest.h` file from source tarball anymore because we are using brewed `doctest`.  If that doesn't matter enough to warrant a revision bump then we can just drop it.

Reply:
I assume `doctest.h` wasn't used outside Homebrew's test, so shouldn't be impacting anything for users.

Since `doctest` is also MIT licensed, I don't think there is any problem with our previous distribution and we can probably avoid revision bump. Could rebuild bottles if wanted.

Reply:
Yes, I doubt any users are going to miss `doctest.h`.